dotnet/src/Skills/Skills.OpenAPI/Skills.OpenAPI.csproj

```diff
@@ -17,7 +17,9 @@
 
   <ItemGroup>
     <None Remove="Skills\AzureKeyVaultSkill\openapi.json" />
+    <None Remove="Skills\GitHubSkill\openapi.json" />
     <EmbeddedResource Include="Skills\AzureKeyVaultSkill\openapi.json" />
+    <EmbeddedResource Include="Skills\GitHubSkill\openapi.json" />
```

Comment:
Let's not perpetuate a pattern of bringing in OpenAPI definitionas to the OpenAPISkill project. We should be able to keep all of this within CopilotChat.

Reply:
Removed this and moved spec to CopilotChat